Mathlib/SetTheory/Cardinal/Finite.lean

```diff
@@ -337,14 +338,42 @@ theorem card_le_one_iff_subsingleton (α : Type*) : card α ≤ 1 ↔ Subsinglet
   rw [← le_one_iff_subsingleton]
   simp [card]
 
+lemma card_eq_one_iff_unique {α : Type*} : card α = 1 ↔ Nonempty (Unique α) := by
+  rw [unique_iff_subsingleton_and_nonempty α, le_antisymm_iff, one_le_iff_ne_zero]
+  exact and_congr (card_le_one_iff_subsingleton α) (card_ne_zero_iff_nonempty α)
+
 theorem one_lt_card_iff_nontrivial (α : Type*) : 1 < card α ↔ Nontrivial α := by
   rw [← Cardinal.one_lt_iff_nontrivial]
   conv_rhs => rw [← Nat.cast_one]
   rw [← natCast_lt_toENat_iff]
   simp only [ENat.card, Nat.cast_one]
 
 @[simp]
-theorem card_prod (α β : Type*) : ENat.card (α × β) = .card α * .card β := by
+theorem card_prod (α β : Type*) : ENat.card (α × β) = card α * card β := by
```

Comment:
```suggestion
theorem card_prod (α β : Type*) : card (α × β) = card α * card β := by
```
You're already in the `ENat` namespace, no need to specify it more. Same thing below.